src/gui/transferlistsortmodel.h

```diff
@@ -55,12 +55,14 @@ class TransferListSortModel final : public QSortFilterProxyModel
 
 private:
     int compare(const QModelIndex &left, const QModelIndex &right) const;
+    void saveSubSortColumn();
+    void loadSubSortColumn();
```

Comment:
I would recommend using `CachedSettingValue<>` class instead of these.

Reply:
these functions are actually not used, will remove them.

Reply:
>... instead of these.

Actually I meant the `Preference` class.